www/src/html.js

```diff
@@ -72,6 +72,10 @@ export default class HTML extends React.Component {
             href={`/safari-pinned-tab.svg`}
             color="#5bbad5"
           />
+          <link
+            rel="stylesheet"
+            href="https://unpkg.com/react-instantsearch-theme-algolia@4.0.0/style.min.css"
+          />
```

Comment:
do we need / use this?

Reply:
I do use their styling for the search box, but mostly included it so I could have something that looked decent fairly quickly. Can definitely take this out and do our own.

Reply:
Got it. Didn't realize we were using it. Let's talk about styling in one fell swoop, but for now just add `rel="preload"`